src/components/App.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Add from "./Add";
+import Items from "./Items/items";
+import Search from "./Search";
+import Counter from "./Counter";
+import PropTypes from "prop-types";
+
+class App extends React.Component {
+  render() {
+    return (
+      <div style={{ height: "100%" }}>
```

Comment:
Use Material's Grid instead of using flex property. You will reduce a lot of css.

Reply:
Got it

Reply:
Okay